frontend/src/components/GVendor.vue

```diff
@@ -71,10 +67,7 @@ export default {
       type: Array,
       default: () => [],
     },
-    cloudProviderKind: {
-      type: String,
-    },
-    cloudProfileName: {
```

Comment:
I would not drop cloudProfileName. We can take the cloud profile name info from `Shoot.spec.cloudProfileName`